gcsio/pom.xml

```diff
@@ -166,6 +166,10 @@
       <groupId>com.google.cloud</groupId>
       <artifactId>google-cloud-core</artifactId>
     </dependency>
+    <dependency>
```

Comment:
Why do we need to write directly to Cloud Logging instead of writing to log the same to local file which automatically pushed to Cloud Logging by fluentd?

Reply:
What queries are we talking about?

[Flogger supports structured logging](https://github.com/google/flogger/blob/11fcf148ca74dc098b225fff01ad69a40f5a4dd9/api/src/main/java/com/google/common/flogger/LoggingApi.java#L212-L260) which can be parsed by fluentd as tags/json fields if necessary.

Overall I'm not sure that maintenance cost and provided benefit worth native integration with Cloud Logging APIs.

I think that it could be better how to make it work on querying site, instead of implementing custom logging backend.

Additionally, I thought that for tracing are usually used specialized systems, not logs, like https://opentelemetry.io that we maybe already integrated with?